package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "wfm-tutorial-client",
+  "name": "fh-wfm-tutorial-client",
```

Comment:
Might rename this to `raincatcher-tutorial-client`

`raincatcher` is the upstream open source project name. `fh-wfm` signifies productised.